src/Tizen.NUI/src/public/BaseComponents/FlexContainer.cs

```diff
@@ -27,9 +27,9 @@ namespace Tizen.NUI.BaseComponents
     /// FlexContainer can expand items to fill available free space, or shrink them to prevent overflow.<br />
     /// </summary>
     /// <since_tizen> 3 </since_tizen>
+    [Obsolete("Deprecated in API8, Will be removed in API11, " + "Please use FlexLayout instead!")]
```

Comment:
According to our deprecation policy when you deprecated API level N, we maintain it in N+1 and remove it in N+2, therefore it should be "Deprecated in API8, will be removed in API10" instead.

Reply:
Done.